openapi/definitions.yaml

```diff
@@ -637,6 +637,33 @@ ServiceTuple:
   required:
     - service_id
     - version
+UserDataProcessing:
+  type: object
+  description: |-
+    A representation of a single user data processing request
+  properties:
+    id:
+      type: string
+      minLength: 1
+    fiscalCode:
+      $ref: "#/FiscalCode"
+    choice:
+      $ref: "#/UserDataProcessingChoice"
+    status:
+      $ref: "#/UserDataProcessingStatus"
+    createdAt:
```

Comment:
properties must be snake_cased into openapi specs (created_at, updated_at, fiscal_code, ...)